css/README.md

```diff
@@ -410,3 +413,47 @@ When using HSL or RGB notation, always add a single space after a comma (,) and
 We prefer using mixins over functions. If a function is absolutely necessary, document where it is used and why it wasn't possible to create a mixin.
 
 **[⬆ back to top](#table-of-contents)**
+
+## CSS Modules
```

Comment:
Sadly there isn't much about best practices concerning css modules in the web. I just thought about some things by myself ...